srcpkgs/hashcash/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'hashcash'
+pkgname=hashcash
+version=1.23
+revision=1
+archs="~*-musl"
+build_style=gnu-makefile
+hostmakedepends="make"
+short_desc="Hashcash proof-of-work algorithm"
+maintainer="Kirill Che. <g4s8.public@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://www.hashcash.org"
+distfiles="http://www.hashcash.org/source/hashcash-${version}-pr.tgz"
+checksum=03a85b6a8a84be93f76a7297fccd5f61dcb8255de25f3fbd77c26be3ef9be999
+make_build_args+=" hashcash docs"
+make_install_args+=" install"
```

Comment:
You should just set these, they don't have an initial value. Also, put this directly below `build_style`.